server/src/test/java/org/opensearch/indices/replication/SegmentReplicationTargetServiceTests.java

```diff
@@ -127,22 +141,33 @@ public void testAlreadyOnNewCheckpoint() {
         verify(spy, times(0)).startReplication(any(), any(), any());
     }
 
-    public void testShardAlreadyReplicating() {
-        SegmentReplicationTargetService spy = spy(sut);
+    public void testShardAlreadyReplicating() throws InterruptedException {
+        SegmentReplicationTargetService serviceSpy = spy(sut);
         // Create a separate target and start it so the shard is already replicating.
```

Comment:
This comment seems out of place now since the replication is not started until multiple lines later. Consider removing it